fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1100,7 +1100,7 @@ public void loadImage(String imageDir) throws IOException, DdlException {
         DataInputStream dis = new DataInputStream(new BufferedInputStream(new FileInputStream(curFile)));
 
         long checksum = 0;
-        long remoteChecksum = 0;
```

Comment:
When does an empty file occur

Reply:
We did not check validation after downloading image. If in some way, the image file is empty, the image will be download successfully, loaded into memory successfully. This is not a real case, but could be real.